src/Microsoft.AspNet.Mvc.Razor/RazorTextWriter.cs

```diff
@@ -51,6 +52,8 @@ public override Encoding Encoding
 
         private TextWriter TargetWriter { get; set; }
 
+        private IHtmlEncoder HtmlEncoder { get; set; } = new HtmlEncoder();
```

Comment:
Are ```HtmlEncoders``` both cheap to new and stateful? If not **both**, perhaps we should cache a static Instance for defaulting? Even if they are stateful, but still expensive to new, perhaps a lazy creation?